ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Docker/tasks/debian.yml

```diff
@@ -0,0 +1,48 @@
+---
+- name: Add Docker GPG apt Key for Debian ( < 10 )
+  apt_key:
+    url: https://download.docker.com/linux/debian/gpg
+    state: present
+  when:
+    - ansible_distribution_major_version|int <10
+
+- name: Add Docker repo for Debian x86_64
+  apt_repository:
+    repo: "deb [arch=amd64] https://download.docker.com/linux/debian {{ ansible_distribution_release }} stable"
+    state: present
+  when:
+    - ansible_distribution_major_version|int <10
+    - ansible_architecture == "x86_64"
+
+###################################################################################################################
+- name: Add Docker GPG apt key for Raspbian
+  apt_key:
+    url: https://download.docker.com/linux/debian/gpg
+    state: present
+  when:
+    - ansible_architecture == "armv7l"
+
+- name: Add Docker repo for Raspbian arm7vl
+  apt_repository:
+    repo: "deb https://download.docker.com/linux/raspbian {{ ansible_distribution_release }} stable"
+    state: present
+  when:
+    - ansible_architecture == "armv7l"
+
+###########################################################################################
+- name: Install Docker prerequisites for Debian
+  apt:
+    pkg: "{{ item }}"
+    state: latest
+  with_items:
+    - ca-certificates
+    - curl
+    - gnupg
+    - lsb-release
+  tags:
+    # TODO: Package installs should not use latest
```

Comment:
This comment doesn't seem to make sense here?

Reply:
i was thinking to have all "TODO" so we can someday set fixed version instead of using a floating "latest" which I feel kinda difficult to track.